packages/backend/src/controllers/gitIntegrationController.ts

```diff
@@ -83,8 +84,8 @@ export class GitIntegrationController extends BaseController {
         @Path() projectUuid: string,
         @Body()
         body: {
-            customMetrics: string[];
-            quoteChar: `"` | `'`; // to be used in the yml dump options
+            customMetrics: AdditionalMetric[];
```

Comment:
changed the api for this endpoint, after https://github.com/lightdash/lightdash/pull/13493 , it should not be used anywhere

Afaik we are only going to use it to create 1 PR per custom metric in the explore UI. But I made it so it can also take a list, in case we want to do it by chart, or push all of them in settings.